doc/user/bgp.rst

```diff
@@ -2921,6 +2921,23 @@ BGP Extended Communities in Route Map
 
    This command sets colors values.
 
+:rfc:`9256`.
+
+``CO:COLOR``
+   This is a format to define colors value. ``CO`` part is always 00 (default),
+   it can be used to support the requirements of Color-Only steering when using 
+   a Null Endpoint in the SR-TE Policy as specified in Section 8.8 of [RFC9256].
+   The below shows in detail what the different combinations of ``CO`` bits can
+   match on to for the purpose of determining what type of SR-TE Policy Tunnel
+   a BGP route can resolve over, and it also shows the order for resolving the
+   BGP route if there are different tunnels.
+   - ``00`` Can match on a specific endpoint only which should be the nexthop
+     of the route(Default Setting).
+   - ``01`` Can match on a specific endpoint or a null endpoint.
+   - ``10`` Can match on a specific endpoint, null endpoint or any endpoint.
+   - ``11`` Reserved for future use and shuould not be used.
+
```

Comment:
- typo in title: extcommunity color
- can you specify that there is no control yet on the CO bits values from an incoming BGP update.
In other words, I can setup whatever SRTE policy, only the color value will be used, not the CO bits.

Reply:
Yes, only color will be used, it will be control by yang.